xls2xml/tests/data/T2D_xls2xml_v1.conf

```diff
@@ -0,0 +1,83 @@
+#Configuration for defining which fields in which worksheets to be parsed in an Excel file
+#
+#At the top level, "worksheets" defines a list of worksheets (titles) to be parsed.
+#Then for each worksheet, you define which fields to be parsed from this worksheet.
+#These fields are either "required" or "optional". "required" fields must be found
+#in the worksheet so that this worksheet is valid. Otherwise this worksheet is not
+#valid. The field names must match those values in the header row in the worksheet.
+#
+#This is a YAML file
+
+# List of worksheets to be parsed.
+worksheets:
+  #- Project
+  #- Cohort
+  #- Phenotype_Trait
+  - Sample
+  #- Attribute Mapping
+  - Analysis
+  #- File
+
+# List of fields for worksheet "Sample"
+Sample: # worksheet title itself
+  # List of "required" fields
+  required:
+    - Sample_ID
+    - Subject_ID
+    - Geno_ID
+    - Phenotype
+    - Gender
+    - Analysis_alias
+    - Cohort ID
+    - Ethnicity
+
+  # List of "optional" fields
+  optional:
+    - Ethnicity Description
+    - T2D
+    - Case_Control
+    - Description
+    - Center_name
+    - Hispanic or Latino; of Spanish origin
+    - Age
+    - Year of Birth
+    - Year of first visit
+    - Cell Type
+    - Maternal_id
+    - Paternal_id
+    - Novel Attributes
+
+  data_type:
+    T2D: int
```

Comment:
In the example excel sheet, it is an integer 0 or 1. It is probably easier to use 0 and 1 too because that is more consistent across different file formats than using `true` and `false`.